applications/luci-app-acme/htdocs/luci-static/resources/view/acme/acme.js

```diff
@@ -665,6 +657,45 @@ function _collectDdnsDomains() {
 	return ddnsDomains;
 }
 
+function _importDdns(ddnsDomains) {
+	alert(_('After import check the added domain certificate configurations.'));
```

Comment:
`alert()` is not a good way to send messages to the uses. Pretty sure luci has some way of adding messages, please use that instead.

Reply:
Please remove this `alert()`